### PR TITLE
Styling changes / Responsive Twitch 

### PR DESCRIPTION
**Pull request by ianmah**
## things
- Mainly update colours *brand refresh*
- And make twitch iframe responsive

378895f04fc90b47b59c552bdf09ef05
5014082d3165a46e1dae16995cf64855

### CODE REVIEW

File: src/components/Typography.js [hunk 1]
```diff
@@ -27,15 +27,15 @@ export const H3 = styled.h3`
 
 export const P = styled.p`
   ${text};
-  ${p => (p.highlight && 'color: ' + p.theme.colors.link)};
+  ${p => (p.highlight && 'color: ' + p.theme.colors.highlight)};
```
Comment (jackyzha0) [inline src/components/Typography.js:30]: can we keep this consistent with the other ones and use template literals here too

File: src/components/Typography.js [hunk 2]
```diff
@@ -27,15 +27,15 @@ export const H3 = styled.h3`
...
   margin: 0;
 `
 
 export const A = styled.a`
   cursor: pointer;
   text-decoration: none;
-  color: ${p => (p.selected ? p.theme.colors.linkHover : p.theme.colors.link)};
-  transition: color 0.3s cubic-bezier(.25,.8,.25,1);
+  color: ${p => p.theme.colors.link};
+  transition: all 0.5s cubic-bezier(.25,.8,.25,1);
```
Comment (jackyzha0) [inline src/components/Typography.js:38]: why is this `all`? what else are we transitioning

Comment (ianmah) [reply src/components/Typography.js:38]: color and background-color

**Review by jackyzha0 — APPROVED** [review]
🔋 lgtm chief

**Pull request merged**